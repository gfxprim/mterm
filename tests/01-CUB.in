2 5
a\e[D\e[D
