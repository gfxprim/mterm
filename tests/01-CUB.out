 --
|a |
|  |
|  |
|  |
|  |
 --
size 5x2 cursor 0x0
